lib/clone_kit/rules/except.rb

```diff
@@ -2,20 +2,24 @@
 
 module CloneKit
   module Rules
+    #
+    # Removes attributes defined by an array
+    # of keys
```

Comment:
:neckbeard: The line breaks seem arbitrary.

Reply:
I don't have an auto-formatter for comments, so they kind of are.

Reply:
I'll clean this one up though. Seems a little short.